cves/kernel/CVE-2020-12652.yml

```diff
@@ -332,8 +348,9 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: true
+  note: | 
+    Sign offs on the commits.
```

Comment:
This is super vague. Most people reading this wouldn't have any idea what a sign off really is. If possible, could you go a little more in depth on how you came to this conclusion?

Reply:
I went into slightly more detail about where the sign offs were located, as well as defined what a sign off is. Will be in my next commit.